gensim/test/scripts/word2vec2tensor.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
```

Comment:
`from gensim.scripts.word2vec2tensor import word2vec2tensor` should work fine (without this file).